website/docs/cli/commands/apply.mdx

```diff
@@ -7,105 +7,86 @@ description: >-
+[planning options](/cli/commands/plan#planning-options) to customize how Terraform will create the plan.
 
-Terraform will propose the plan to you and prompt you to approve it before
-taking the described actions, unless you waive that prompt by using the
-`-auto-approve` option.
+You can pass the `-auto-approve` option to instruct Terraform to apply the plan without asking for confirmation.
 
--> **Tip:** When you run `terraform apply` without a saved plan file, you can use all of the [planning modes](/cli/commands/plan#planning-modes) and
-[planning options](/cli/commands/plan#planning-options) available for `terraform plan`. This lets you customize how Terraform will create the plan.
+!> **Warning:** If you use `-auto-approve`, we recommend making sure that no one can change your infrastructure outside of your current workflow. This minimizes the risk of unpredictable changes and configuration drift.
 
 ### Saved Plan Mode
 
-If you pass the filename of a previously-saved plan file, `terraform apply`
-performs exactly the steps specified by that plan file. It does not prompt for
-approval; if you want to inspect a plan file before applying it, you can use
-[`terraform show`](/cli/commands/show).
+When you pass a [saved plan file](/cli/commands/plan#out-filename) to `terraform apply`, Terraform takes the actions in the saved plan without prompting you for confirmation. You may want to use this two-step workflow when [running Terraform in automation](https://learn.hashicorp.com/tutorials/terraform/automate-terraform?in=terraform/automation&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS).
+
+Use [`terraform show`](/cli/commands/show) to inspect a saved plan file before applying it.
 
-When using a saved plan, none of the planning modes or planning options linked
-above are supported; these options only affect Terraform's decisions about which
+When using a saved plan, you cannot specify any additional planning modes or options. These options only affect Terraform's decisions about which
 actions to take, and the plan file contains the final results of those
 decisions.
 
 ### Plan Options
 
-When run without a saved plan file, `terraform apply` supports all of `terraform
-plan`'s planning modes and planning options. For details, see:
+Without a saved plan file, `terraform apply` supports all planning modes and planning options available for `terraform plan`.
 
-* [Planning Modes](/cli/commands/plan#planning-modes)
-* [Planning Options](/cli/commands/plan#planning-options)
+-  **[Planning Modes](/cli/commands/plan#planning-modes):** These include `-destroy`, which creates a plan to destroy all remote objects, and `-refresh-only` which creates a plan to update Terraform state and root module output values.
```

Comment:
```suggestion
-  **[Planning Modes](/cli/commands/plan#planning-modes):** These include `-destroy`, which creates a plan to destroy all remote objects, and `-refresh-only`, which creates a plan to update Terraform state and root module output values.
```

Reply:
By the way, I like the additional context for what these things are! 👍🏼

Reply:
thank you :)